cockatrice/src/gamesmodel.cpp

```diff
@@ -314,6 +314,46 @@ void GamesProxyModel::setMaxPlayersFilter(int _maxPlayersFilterMin, int _maxPlay
     invalidateFilter();
 }
 
+void GamesProxyModel::setMaxGameAge(const int _maxGameAge)
+{
+    maxGameAge = static_cast<MaxGameAge>(_maxGameAge);
+    invalidateFilter();
+}
+
+int GamesProxyModel::getMaxGameAgeInSeconds(MaxGameAge maxGameAge)
+{
+    switch (maxGameAge)
+    {
+        case FIVE_MINUTES:
```

Comment:
Just questioning, why not store these as our enum values? (FIVE_MINUTES = 300, TEN_MINUTES = 600, etc). Would remove the need to translate on the fly, and it guarantees no collisions

Reply:
why not simply use qt's already preexisting time classes instead of copying some enums from a bad implementation 5 years ago? (25dbfb37f based on 93ab9f969 which introduced these weird values)

Reply:
I fixed it

Reply:
> 
> 
> I fixed it

--> #4095. Thanks!